settings/class.hooks.php

```diff
@@ -256,9 +256,9 @@ public function ProfileController_Best_Create($Sender, $UserReference = '', $Use
    * @param ProfileController $Sender
    */
   public function ProfileController_AddProfileTabs_Handler($Sender) {
-    if(is_object($Sender->User) && $Sender->User->UserID > 0) {
+    /*if(is_object($Sender->User) && $Sender->User->UserID > 0) {
```

Comment:
Is there a reason you left this in commented?

Reply:
If the code should not be there, the entire hook should be removed. In this patch how does a user get to their profile best content?

Reply:
Yes remove comments, i will search another solution

Reply:
Feel free to play with it and push more commits to this branch

Reply:
@pioc92 That page already has `robots: noindex/noarchive`. This is not a SEO problem. You can just hide the link with CSS if you don't like it.

Reply:
@bleistivt when you crawl the forum, the crawler can't stop because of this link. Try it with screaming frog!